pkg/machine/e2e/config_windows_test.go

```diff
@@ -12,7 +12,10 @@ import (
 
 const podmanBinary = "../../../bin/windows/podman.exe"
 
-func getDownloadLocation(_ machine.VirtProvider) string {
+func getDownloadLocation(p machine.VirtProvider) string {
+	if p.VMType() != machine.WSLVirt {
```

Comment:
Ed always yells at me about negative conditions being hard to read.  After reading the commit message, understanding this is about HyperV, I had some minor astonishment seeing changes to something with `WSL` in the name.  

Is it done b/c there's no (for example) `machine.HypervVirt` to check?  Would it be a PITA to add something like that?   If so, maybe adding a comment like `// HyperV only` would offer some help to tired eyes?

Otherwise I think this is kind of minor, so feel free to ignore.